### ISSUE TITLE
click not firing mouseover event

### ISSUE THREAD

**Issue by jkeruzec** [cypress-io/cypress#2928]
### Current behavior:

I trigger some react-contextify menu with `.trigger('contextmenu')`
My menu is displaying OK

![image](https://user-images.githubusercontent.com/3499632/49786289-8f05ba00-fd23-11e8-9025-94ebbd687b50.png)

Then I select the div element where an onClick event is attached.
`cy.get('.react-contexify__item').debug().click();`

I tried to use also "force: true" as an option.

But my action is not triggered. Nothing happens.

With debug() option activated, I trigger manually the click() event and my Material UI dialog is displayed.

``` javascript
------------------------ Debug Info ------------------------
cypress_runner.js:63678 Command Name:     get
cypress_runner.js:63678 Command Args:     [".react-contexify__item"]
cypress_runner.js:63678 Current Subject:  jQuery.fn.init [div.react-contexify__item, prevObject: jQuery.fn.init(1), context: document, selector: ".react-contexify__item"]
subject.click()
MegaTrendPage.jsx:316 hello!
jQuery.fn.init [div.react-contexify__item, prevObject: jQuery.fn.init(1), context: document, selector: ".react-contexify__item"]
```
Source for div menu is like this: 

``` javascript
  return (
      <div
        className={cssClasses}
        style={style}
        onClick={this.handleClick}
        role="presentation"
      >
        <div className={styles.itemContent}>{children}</div>
      </div>
    );
```

### Desired behavior:

I trigger click event and dialog is shown...

### Versions

Cypress Version used : 3.1.3
Chrome 70.0.3538.110 
React 16.6.3
react-contexify : 3.0.3

**Comment by jennifer-shehane** [comment]
Could you provide a reproducible example of this? With test code + application code.

**Comment by Bkucera** [comment]
@jkeruzec I was able to reproduce, thank you. We are aware of this issue because we do not yet send mouseover/pointerover events during a click.

We'll have a fix in either 3.1.3 or 4.0

in the meantime, you can do this: 
```
cy.get('.react-contexify__item__data')	
  .trigger('mouseover').click('left')
  .then(() => {
	 expect(stub).to.be.calledWith('Hello bug !');
  });
```

**Comment by jkeruzec** [comment]
Workaround works beautifully ! 

Thanks for that !
